Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -3943,6 +3943,74 @@ function Invoke-CrmWhoAmI{
     return $result
 }
 
+function Invoke-CrmAction {
+# .ExternalHelp Microsoft.Xrm.Data.PowerShell.Help.xml
+    [OutputType([hashtable])]
+    [OutputType([Microsoft.Xrm.Sdk.OrganizationResponse], ParameterSetName="Raw")]
+    param (
+        [Microsoft.Xrm.Tooling.Connector.CrmServiceClient]
+        $conn,
+
+        [Parameter(
+            Position=1,
+            Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Name,
+
+        [Parameter(ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [Microsoft.Xrm.Sdk.EntityReference]
+        $Target,
+
+        [Parameter(Position=2)]
+        [ValidateNotNullOrEmpty()]
+        [hashtable]
+        $Parameters,
+
+        [Parameter(ParameterSetName="Raw")]
+        [switch]
+        $Raw
```

Comment:
I see your point and I agree with you. I just didn't get it from the parameter name. Maybe RawResult? Not sure which word works but anyway it is good to have that option.